client/src/components/Comment.vue

```diff
@@ -1,65 +1,74 @@
 <template>
```

Comment:
Cool wäre es, wenn ich einen Link (http[s]?://.*) in den Kommentar packe, das dieser automatisch in ein <a href> gepackt wird.

Reply:
genau so ist es ja aktuell auch. Allerdings nur beim rendern. ansonsten müssten wir nen markdown editor einbauen.